benchexec/runexecutor.py

```diff
@@ -656,7 +663,7 @@ def _setup_ulimit_time_limit(self, hardtimelimit, cgroups):
     def execute_run(self, args, output_filename, stdin=None,
                    hardtimelimit=None, softtimelimit=None, walltimelimit=None,
                    cores=None, memlimit=None, memory_nodes=None,
-                   environments={}, workingDir=None, maxLogfileSize=None,
+                   environments={}, energy_measurement=None, workingDir=None, maxLogfileSize=None,
```

Comment:
There is old code using some script `read-energy.sh`, so I wanted to make energy measurements interchangeable. It was kind of arbitrary, we can also change that.